packages/terra-form-input/src/Input.module.scss

```diff
@@ -87,15 +87,15 @@
 
     &:focus {
       background-color: var(--terra-form-input-error-focus-background-color, #fff);
-      background-size: var(--terra-form-input-error-focus-keyboard-animation-size);
+      background-size: var(--terra-form-input-error-focus-keyboard-animation-background-size);
```

Comment:
nit: I'd drop `keyboard-animation` from this variable

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/21cdfb37f22d61dfd8a9b52b1f3ab811e65ccd4b#diff-462afbdb742ea35bac7504edbfd87d7dR90